assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WidgetAreaRenderer component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { string } from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../datastore';
+import Widget from './WidgetRenderer';
+
+const { useSelect } = Data;
+
+const WidgetAreaRenderer = ( { slug } ) => {
+	const widgetArea = useSelect( ( select ) => select( STORE_NAME ).getWidgetArea( slug ) );
+	const widgets = useSelect( ( select ) => select( STORE_NAME ).getWidgets( slug ) );
+
+	return (
+		<div className={ classnames( 'mdc-layout-grid', 'googlesitekit-widget-area', `googlesitekit-widget-area-${ widgetArea.slug }`, `googlesitekit-widget-area__${ widgetArea.style }` ) }>
```

Comment:
Actually, two of these classes are off, via BEM they should be:
* `googlesitekit-widget-area--${ widgetArea.slug }`
* `googlesitekit-widget-area--${ widgetArea.style }`
Since these are modifiers.